app/src/organisms/ModuleWizardFlows/AttachProbe.tsx

```diff
@@ -135,19 +146,30 @@ export const AttachProbe = (props: AttachProbeProps): JSX.Element | null => {
       </InProgressModal>
     )
 
-  const bodyText =
-    attachedPipetteChannels === 8 || attachedPipetteChannels === 96 ? (
+  const bodyText = (
+    <>
       <Trans
         t={t}
-        i18nKey={i18nKey}
+        i18nKey={'pipette_wizard_flows:install_probe'}
+        values={{ location: probeLocation }}
         components={{
           strong: <strong />,
           block: <StyledText css={BODY_STYLE} />,
         }}
       />
-    ) : (
-      <StyledText css={BODY_STYLE}>{t('install_probe')}</StyledText>
-    )
+      {wasteChuteConflict && (
+        <Banner
+          type={isWasteChuteOnDeck ? 'error' : 'warning'}
+          size={isOnDevice ? '1.5rem' : SIZE_1}
```

Comment:
nit
We stopped using `SIZE_number` for the future Design system unification.

```suggestion
          size={isOnDevice ? '1.5rem' : '1rem'}
```